cfme/configure/configuration/__init__.py

```diff
@@ -515,6 +515,47 @@ def update(self):
         self.appliance.server_details_changed()
 
 
+class VMwareConsoleSupport(Updateable, Pretty, Navigatable):
+    """
+    This class represents the "VMware Console Support" section of the Configuration page.
+    Note this is to support CFME 5.8 and beyond functionality.
+
+    Args:
+        console_type: VMware VMRC Plugin, VNC or VMware WebMKS
+
+    Usage:
+
+        vmware_console_support = VMwareConsoleSupport(console_type="VNC")
+        vmware_console_support.update()
+
+    """
+    vmware_console_form = Form(
+        fields=[
+            ('console_type', AngularSelect("console_type")),
+        ]
+    )
+    pretty_attrs = ['console_type']
+
+    def __init__(self, console_type=None, appliance=None):
+        assert (console_type), "You must provide at least one param!"
```

Comment:
Just remove the default (None) from `console_type` and it becomes a required parameter. python will raise an exception if the class is instantiated without it.